File: 프로그래머스/0/181927. 마지막 두 원소/마지막 두 원소.py
def solution(num_list):
    a = num_list[-1]
    b = num_list[-2]
    if num_list[-1] > num_list[-2]:
        num_list.append(int(a-b))
    else:
        num_list.append(int(a)*2)
    return num_list

File: 프로그래머스/0/181927. 마지막 두 원소/마지막 두 원소.js
function solution(num_list) {
  const l = num_list.length;
  return [...num_list, num_list[l - 1] > num_list[l - 2] 
    ? num_list[l - 1] - num_list[l - 2] 
    : num_list[l - 1] * 2];
}
